backends/advanced/src/advanced_omi_backend/database.py

```
@@ -706,6 +738,50 @@ async def activate_transcript_version(self, conversation_id: str, version_id: st
             logger.info(f"Activated transcript version {version_id} for conversation {conversation_id}")
         return result.modified_count > 0
 
+    async def update_transcript_version(
+        self, 
+        conversation_id: str, 
+        version_id: str, 
+        transcript: str = None, 
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)